validator/build.gradle

```diff
@@ -55,6 +59,10 @@ dependencies {
     implementation 'com.amazonaws:aws-java-sdk-cloudwatch'
     implementation 'com.amazonaws:aws-java-sdk-xray'
 
+    // https://mvnrepository.com/artifact/com.github.awslabs/aws-request-signing-apache-interceptor
+    implementation 'com.github.awslabs:aws-request-signing-apache-interceptor:b3772780da'
+    implementation files('libs/AWSPrometheusJavaClient-1.11.x.jar')
```

Comment:
when will this lib to be available on maven?

Reply:
I forgot to remove this jar (we don't actually use it in the code and it's not actually in this PR).